src/lib.rs

```diff
@@ -792,6 +792,30 @@ macro_rules! impl_ranged {
                 <Self as $crate::traits::RangeIsValid>::ASSERT;
                 Self::new_saturating(self.get().saturating_pow(exp))
             }
+
+            /// Wrapping integer addition. Computes `self + rhs`, wrapping around the numeric
+            /// bounds.
+            #[must_use = "this returns the result of the operation, without modifying the original"]
+            #[inline]
+            pub fn wrapping_add(self, rhs: $internal) -> Self {
```

Comment:
Can the method be made `const fn` without a ton of effort?

Reply:
Fix: [dd5ed24](https://github.com/jhpratt/deranged/pull/9/commits/dd5ed246aa63c17ae7720e84a4ec7330aede3f85)